src/REAXFF/fix_reaxff_species.cpp

```diff
@@ -239,7 +241,14 @@ FixReaxFFSpecies::FixReaxFFSpecies(LAMMPS *lmp, int narg, char **arg) :
       // rate limit when deleting molecules
     } else if (strcmp(arg[iarg], "delete_rate_limit") == 0) {
       if (iarg + 3 > narg) utils::missing_cmd_args(FLERR, "fix reaxff/species delete_rate_limit", error);
-      delete_Nlimit = utils::numeric(FLERR, arg[iarg+1], false, lmp);
+      delete_Nlimit_varid = -1;
+      if (strncmp(arg[iarg+1],"v_",2) == 0) {
+        delete_Nlimit_varid = input->variable->find(&arg[iarg+1][2]);
```

Comment:
The usual way to handle variables in fixes is to make a copy of the variable name and then check again and look up the index in `Fix::init()`. Variables could be deleted between the definition of the fix and a run, when it is used.